aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -39,6 +39,15 @@ void exp_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+void exp2_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "exp2_cuda", [&]() {
+    gpu_kernel(iter, [] GPU_LAMBDA(scalar_t a) -> scalar_t {
+      auto result = ::exp2(a);
```

Comment:
code golf: `return ::exp2(a);`

This would also be consistent with the `exp` kernel implementation.

Reply:
Ah. Right! Was trying something, failed to clean up. Thanks!